Makefile.dep

```diff
@@ -466,11 +466,13 @@ ifneq (,$(filter stdio_uart_rx,$(USEMODULE)))
   USEMODULE += stdio_uart
 endif
 
-ifneq (,$(filter stdio_uart,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_uart
+ifneq (stdio_rtt,$(filter stdio_%,$(USEMODULE)))
+  ifneq (,$(filter stdio_%,$(USEMODULE)))
+    FEATURES_REQUIRED += periph_uart
```

Comment:
This required periph_uart in all cases except stdio_rtt so this includes `stdio_null`, `stdio_cdc_acm` but doesn't make sense.
I would decouple this:
```mk
ifneq (,$(filter stdio_uart,$(USEMODULE)))
  FEATURES_REQUIRED += periph_uart
endif

ifneq (stdio_rtt,$(filter stdio_%,$(USEMODULE)))
  # stdio_rtt cannot be used when stdio_uart/stdio_cdc_acm are loaded
  DISABLE_MODULE += stdio_rtt
endif
```

Reply:
Oops, this was not intentional.

Reply:
Re-reading my proposal, this is not better but even worse...

Reply:
I fixed it as I thought it was right.

Reply:
Latest commit should be the real thing now :sweat_smile: